integration/config.ts

```diff
@@ -3,6 +3,7 @@ import { LoggerInstance, LogLevel, makeAccounts } from '../src/utils'
 
 LoggerInstance.setLevel(LogLevel.Error)
 
+const logLevel = Number(process.env['LOG_LEVEL'])
```

Comment:
won't this fail if the LOG_LEVEL is not defined. Maybe we should add a default

Reply:
it uses `error` by default but adding a 1 (`warn`) as default value if `LOG_LEVEL` is not defined